### PR TITLE
Added image crop to crop bot image

### LINKED ISSUE #3296: Use react image crop for cropping bot image
**Expected Behaviour**

Set react image crop to crop the bot image before uploading in **Create bot page**.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Dilshaad21**
Fixes #3296

Changes:
- Used image crop modal to crop the bot image before upload.

Demo Link: https://pr-3297-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 

Screenshot from 2020-01-16 22-00-06

### CODE REVIEW

**Comment by Dilshaad21** [comment]
@akshatnitd updated `Dialog` title dynamically.

**Comment by anishagg17** [comment]
@Dilshaad21 i think instead of upload button on the dialog a crop button should be there

File: src/components/cms/SkillCreator/SkillWizard.js [hunk 1]
```diff
@@ -939,24 +940,35 @@ class SkillWizard extends Component {
...
-    const pattern = /^::image\s(.*)$/m;
-    code = code.replace(pattern, `::image images/${imageUrl}`);
-    payload = { ...payload, file, imageUrl, code, image };
-    actions.setSkillData(payload);
+    let image = window.URL.createObjectURL(file);
+    let imageName = file.name;
+
+    await actions.openModal({
+      modalType: 'crop',
+      title: <div>Crop {this.isBotBuilder ? 'bot' : 'skill'} image</div>,
```
Comment (akshatnitd) [inline src/components/cms/SkillCreator/SkillWizard.js]: Why do we need to wrap it in `<div>`>?

Comment (Dilshaad21) [reply src/components/cms/SkillCreator/SkillWizard.js]: Used template-literals

**Pull request merged**